spec/features/idv/hybrid_flow_spec.rb

```diff
@@ -23,9 +25,10 @@
 
     perform_in_browser(:desktop) do
       user = sign_in_and_2fa_user
-      complete_doc_auth_steps_before_send_link_step
-      fill_in :doc_auth_phone, with: '415-555-0199'
-      click_idv_continue
+      complete_doc_auth_steps_before_upload_step
+      fill_in :doc_auth_phone, with: ''
+      fill_in :doc_auth_phone, with: phone_number
```

Comment:
Why is :doc_auth_phone being filled twice? I see the same code on lines 73-74 and 90-91. Seems like we shouldn't have to make it blank before filling it in for tests to work.

Reply:
The default behavior in the application is to pre-populate the MFA number the user used for signing in. The tests look for a specific secondary number, and therefore we need to clear the field first

Reply:
Could we leave a code comment to that effect or create a named helper, since it seems like it could be prone to confusion for future maintainers.